exercise/templates/exercise/_duplicate_submission_modal.html

```diff
@@ -0,0 +1,48 @@
+{% load i18n %}
+<div
+    class="modal"
+    id="duplicate-submission-modal"
+    tabindex="-1"
+    role="dialog"
+    aria-labelledby="duplicate-submission-modal-label"
+>
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button
+                    type="button"
+                    class="close"
+                    data-dismiss="modal"
+                    aria-label="close"
+                >
+                    <span aria-hidden="true">&times;</span>
+                </button>
+                <h4 class="modal-title" id="duplicate-submission-modal-label">
+                    {% translate "DUPLICATE_SUBMISSION_TITLE" %}
+                </h4>
+            </div>
+            <div class="modal-body">
+                {% blocktranslate trimmed %}
+                    DUPLICATE_SUBMISSION
+                {% endblocktranslate %}
```

Comment:
If the string contains no parameters, then use `{% translate %}` instead of `blocktranslate`.